owlkettle/adw.nim

```diff
@@ -818,8 +849,20 @@ when AdwVersion >= (1, 2) or defined(owlkettleDocs):
     issueUrl: string
     website: string
     copyright: string
-    license: string
-    
+    license: string ## Sets the license as a custom text if it can’t be set via licenseType (when set licenseType will be set to GTK_LICENSE_CUSTOM).
+    licenseType: LicenseType ## Sets the license for from a list of known license.
+    legalSections: seq[LegalSection] ## Adds an extra section to the Legal page. Extra sections will be displayed below the application’s own information. The parameters copyright, licenseType and license will be used to present the it the same way as AboutWindow:copyright, AboutWindow:licenseType and AboutWindow:license are for the application’s own information. See those properties for more details. This can be useful to attribute the application dependencies or data.
```

Comment:
Using the GTK doc comments is an issue as 

1. They often do not apply to owlkettle. E.g. in this case copyright, licenseType and license are not parameters to legalSections, as it is a field, not a function. 
2. GTK is LGPL licensed. I am not a lawyer, but from my understanding, as this license also applies to the (doc) comments, using GTK's doc comments would require owlkettle to be relicensed under LGPL (or a compatible license).

Reply:
I adjusted the comments to not be a direct copy of the GTK docs and to better fit owlkettle usage in some cases.